backend/Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM golang:1.16 AS build
 WORKDIR /go/src/github.com/org/repo
 COPY . .
 
-RUN go build -o server .
+# CGO has to be disabled for alpine image
+RUN export CGO_ENABLED=0 && go build -o server .
```

Comment:
Good catch, thanks! 
Can we directly use a `ENV` command instead of adding an export to the `RUN` one? 
```
ENV CGO_ENABLED=0
RUN go build -o server .
```

Reply:
Hi @glours ,

I can not update a new change on this PR for some reasons, it's already moved to #4  Thanks for very quick reply.